infrastructure/main.tf

```diff
@@ -2,6 +2,18 @@ provider "azurerm" {
   version = "=1.42.0"
 }
 
+locals {
+  is_preview  = "${(var.env == "preview" || var.env == "spreview")}"
+  local_env   = "${local.is_preview ? "aat" : var.env}"
+
+  s2s_rg  = "rpe-service-auth-provider-${local.local_env}"
+  s2s_url = "http://${local.s2s_rg}.service.core-compute-${local.local_env}.internal"
+
+  core_app_settings = {
+    S2S_URL = "${local.s2s_url}"
+  }
+}
+
```

Comment:
we have moved to aks not sure why we need this ?

Reply:
will fix